package.json

```diff
@@ -38,7 +38,7 @@
     "node-sass": "^4.5.3",
     "sass-loader": "^6.0.6",
     "vue-loader": "^13.0.2",
-    "vue-template-compiler": "^2.4.2",
+    "vue-template-compiler": "2.4.2",
```

Comment:
If the vue-template-complier is not the same version of vue we may have an error while installing. I remove the "^" to make sure that they are always the same.